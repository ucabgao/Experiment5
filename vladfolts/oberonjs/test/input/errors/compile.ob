MODULE m1;
END m2.