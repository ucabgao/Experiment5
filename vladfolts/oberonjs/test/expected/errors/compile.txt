line 2: original module name 'm1' expected, got 'm2'
